tests/types/types.go

```diff
@@ -112,9 +122,40 @@ func (ps Partitions) AddRemovedNodes(label string, ns []Node) {
 	p.RemovedNodes = append(p.RemovedNodes, ns...)
 }
 
+func (d Disk) SetOffsets() {
```

Comment:
nit: a comment stating what this function does would be nice. Same for `CalculateImageSize` and`Align` below.